set.mm

```diff
@@ -341639,6 +341684,51 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
       KJZWBWAGVGRRGGVBVCVDVESZVHDEVLVNWCVIVJVK $.
   $}
 
+  ${
+    $d A x y z $.  $d F x y $.  $d G x y $.  $d X x y z $.  $d ph x y z $.
+    efabl.1 $e |- F = ( x e. X |-> ( exp ` ( A x. x ) ) ) $.
+    efabl.2 $e |- G = ( ( mulGrp ` CCfld ) |`s ran F ) $.
+    efabl.3 $e |- ( ph -> A e. CC ) $.
+    efabl.4 $e |- ( ph -> X e. ( SubGrp ` CCfld ) ) $.
+    $( The image of a subgroup of the group ` + ` , under the exponential
+       function of a scaled complex number, is an Abelian group.  (Contributed
+       by Paul Chapman, 25-Apr-2008.)  (Revised by Mario Carneiro,
+       12-May-2014.)  (Revised by Thierry Arnoux, 26-Jan-2020.) $)
+    efabl $p |- ( ph -> G e. Abel ) $=
+      ( ccnfld co cfv eqid wcel wceq syl cmul cc cvv vy cress cplusg cbs cv w3a
+      simp1 simp2 csubg subgbas eleqtrrd simp3 caddc jca efgh syl3an1 ressplusg
+      wa cnfldadd 3ad2ant1 oveqd fveq2d ce cmpt mptexg syl5eqel rnexg 3syl cmgp
+      crn cnfldmul mgpplusg 3eqtr3d syl3anc wfo wfn wral fvex rgenw fnmpt ax-mp
+      dffn4 mpbi eqidd wss wf eff adantr cnfldbas subgss sselda mulcld ffvelrnd
+      a1i ralrimiva rnmptss mgpbas ressbas2 foeq123d mpbii cnrng rngabl subgabl
+      cabl crg sylancr ghmabl ) ABUAKFUBLZUCMZEUCMZDXHEXHUDMZEUDMZXKNXLNXINXJNA
+      BUEZXKOZUAUEZXKOZUFZAXMFOZXOFOZXMXOXILZDMZXMDMZXODMZXJLZPAXNXPUGZXQXMXKFA
+      XNXPUHXQAFXKPZYEAFKUIMZOZYFJFKXHXHNZUJQZQZUKXQXOXKFAXNXPULYKUKAXRXSUFZXMX
+      OUMLZDMZYBYCRLZYAYDACSOZYHURXRXSYNYOPAYPYHIJUNBCXMXODFGUOUPYLYMXTDYLUMXIX
+      MXOAXRUMXIPZXSAYHYQJFUMKXHYGYIUSUQQUTVAVBYLRXJYBYCAXRRXJPZXSADVJZTOZYRAYH
+      DTOYTJYHDBFCXMRLZVCMZVDTGBFUUBYGVEVFDTVGVHYSRKVIMZETHKRUUCUUCNZVKVLUQQUTV
+      AVMVNAFYSDVOZXKXLDVODFVPZUUEUUBTOZBFVQUUFUUGBFUUAVCVRVSBFUUBDTGVTWAFDWBWC
+      AFXKYSXLDDADWDYJAUUBSOZBFVQYSSWEYSXLPAUUHBFAXRURZSSUUAVCSSVCWFUUIWGWNUUIC
+      XMAYPXRIWHAFSXMAYHFSWEJSFKWIWJQWKWLWMWOBFUUBSDGWPYSSEUUCHSKUUCUUDWIWQWRVH
+      WSWTAKXDOZYHXHXDOKXEOUUJXAKXBWAJFKXHYIXCXFXG $.
+  $}
+
+  ${
+    $d x y C $.  $d y T $.
+    circgrp.1 $e |- C = ( `' abs " { 1 } ) $.
+    circgrp.2 $e |- T = ( ( mulGrp ` CCfld ) |`s C ) $.
+    $( The circle group ` T ` is an Abelian group.  (Contributed by Paul
+       Chapman, 25-Mar-2008.)  (Revised by Mario Carneiro, 13-May-2014.)
+       (Revised by Thierry Arnoux, 26-Jan-2020.) $)
+    circgrp $p |- T e. Abel $=
```

Comment:
ditto here: the statement that C is a subgroup of the unit group of CC should be exported, this is just a corollary

Reply:
Done in the latest commit!